components/header.tsx

```diff
@@ -6,20 +6,36 @@ import InnerHeader from './innerHeader';
 export const TabIds = {
     HOME: 'home',
     PRODUCTS: 'products',
+    ORDERS: 'orders',
 };
 
 export const TabRoutes = {
     [TabIds.HOME]: '/',
     [TabIds.PRODUCTS]: '/products',
+    [TabIds.ORDERS]: '/orders',
```

Comment:
orders shouldn't be a tabbed route (since we don't want it to appear in the main tabbed navigation).